.github/scripts/build-default-workloads.py

```diff
@@ -10,12 +10,10 @@ def build_default_workloads():
     with prefix('cd {} && source ./env.sh'.format(manager_fsim_dir)), \
          prefix('cd deploy/workloads'):
 
-        # avoid logging excessive amounts to prevent GH-A masking secrets (which slows down log output)
-        with settings(warn_only=True):
-            rc = run("marshal -v build br-base.json &> br-base.full.log").return_code
-            if rc != 0:
-                run("cat br-base.full.log")
-                raise Exception("Building br-base.json failed to run")
+        # Better support very parallel builds under marshal (1024 on FPGA developer AMI)
+        # See https://github.com/firesim/firesim/pull/1132
+        with prefix('ulimit -n 4096'):
+            run("marshal -v build br-base.json")
```

Comment:
Of course I can't repro this on my AWS manager.  I was thinking that it might be a good idea to have ulimit tell us what it was able to set the limit to in case there is something weird in our CI setup that is causing the hardlimit to be set so that we're not actually increasing it to 4096...
```suggestion
        # Better support very parallel builds under marshal (1024 on FPGA developer AMI)
        # See https://github.com/firesim/firesim/pull/1132
        with prefix('ulimit -n 4096'):
            run("ulimit -n && marshal -v build br-base.json")
```